docs/content/latest/architecture/transactions/read-committed.md

```diff
@@ -0,0 +1,785 @@
+---
+title: READ COMMITTED isolation level
+headerTitle: READ COMMITTED isolation level
+linkTitle: READ COMMITTED isolation level
+description: Details about READ COMMITTED isolation level
+aliases:
+  - /architecture/transactions/read-committed/
+menu:
+  latest:
+    identifier: architecture-read-committed
+    parent: architecture-acid-transactions
+    weight: 1152
+isTocNested: true
+showAsideToc: true
+---
+
+## YSQL Read Committed Isolation Level
```

Comment:
This H2 is not required. Bump everything up a level.
```suggestion
```

Reply:
done